internal/converter/internal/otelcolconvert/otelcolconvert_test.go

```diff
@@ -3,12 +3,16 @@
 package otelcolconvert_test
 
 import (
+	"flag"
 	"testing"
 
 	"github.com/grafana/alloy/internal/converter/internal/otelcolconvert"
 	"github.com/grafana/alloy/internal/converter/internal/test_common"
 )
 
+// Set this flag to update snapshots e.g. `go test -v ./interal/converter/internal/otelcolconverter/...` -fix-tests
+var fixTestsFlag = flag.Bool("fix-tests", false, "update the test files with the current generated content")
```

Comment:
What led you to choose -fix-tests over the original commented -update? Upstream OTel does -update, I'm not sure if this is a wider convention or if you're aware of a convention, IMO would like to follow convention if there is one.

Reply:
From here https://github.com/grafana/alloy/pull/3220#discussion_r2048680571. When I added a this for validation it was pointed out that we already had a place that used -fix-tests so I added it to be consistent. We can change that flag but then we should do so in all places